### ISSUE TITLE
fping echo exception

### ISSUE THREAD

**Issue by houbaicen** [schweikert/fping#182]
When I use the fping command about "fping IP -c 6 -q 2>&1"

Its echo is similar to "127.0.0.1 : xmt/rcv/%loss = 5/6/120%, min/avg/max = 0.25/0.35/0.53"

There are two questions, the first is the data is normal but the loss is very abnormal 120%, and the second  is the xmt is 5 but the rcv is 6 but I set the value about packet is 6.

Please help me with this problem.

**Comment by schweikert** [comment]
Can you give more details about how to reproduce this? What fping version are you using?

**Comment by houbaicen** [comment]
Hi David: 
&nbsp; &nbsp; The fping version is "3.5" and i have saved fpings' echo&nbsp; in influxdb, and the problematic data i can see in influxdb is that "source_ip(192.168.134.110),target_ip(192.168.137.126),xmt(5),rcv(6),loss(120),max(0.34),avg(0.28),min(0.16)"
and has much data like this,but only&nbsp;account for a&nbsp;small part of all data.
&nbsp; &nbsp; I has just parsed the data and didn't do any other processing, so the data in influxdb is same as the fpings' echo data.


Thanks for your help!
&nbsp; &nbsp;&nbsp;
&nbsp; &nbsp;&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "schweikert/fping"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月1日(星期六) 中午1:35
收件人:&nbsp;"schweikert/fping"<fping@noreply.github.com&gt;;
抄送:&nbsp;"幸运女神在微笑"<1716337572@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [schweikert/fping] fping echo exception (#182)





 
Can you give more details about how to reproduce this? What fping version are you using?
